website/source/partials/builders/_aws-spot-docs.html.md

```diff
@@ -0,0 +1,26 @@
+-   `spot_instance_types` (array of strings) - a list of acceptable instances
```

Comment:
could `a list of acceptable instance types` be more precise in this case ?

Reply:
👍